### ISSUE TITLE
Offline transaction example fails if you only have this repo in the gopath

### ISSUE THREAD

**Issue by LeeSmet** [rivine/rivine#109]
The import `"github.com/NebulousLabs/entropy-mnemonics"` is vendored in rivine, but the offline-transaction example is not part of rivine, thus go is a bit picky about the import path. Mainly, the offline example does not have a vendor dir by itself, and as a result you must also check out the referenced repo for the offline transaction example to run. Since we can't explicitly reference the vendor directory in the rivine project, there needs to be a different solution

**Comment by GlenDC** [comment]
in worst case you could simply add a soft-link, no? Would break on platforms such as Windows I suppose, so not ideal.

**Comment by robvanmieghem** [comment]
I don't think we use "github.com/NebulousLabs/entropy-mnemonics" any longer  and since it was only used as a seed for generating addresses, I don't think it should be needed for this example.

**Comment by robvanmieghem** [comment]
moved away from backlog since I have a suspicion this is broken because se do not use "github.com/NebulousLabs/entropy-mnemonics" any longer.

**Comment by GlenDC** [comment]
Do you want to take a look at it @robvanmieghem, otherwise I don't mind checking it at some point, would give me an excuse to spend some time on getting to know that example a bit better
